app/views/layouts/_header.html.haml

```diff
@@ -19,7 +19,4 @@
                 = link_to AppConfig.settings.pod_name, root_path, class: "navbar-brand"
 
               .collapse.navbar-collapse#navbar-collapse
-                %ul.nav.navbar-nav.navbar-right
-                  - if AppConfig.settings.enable_registrations? && !current_page?(controller: "/registrations", action: :new)
-                    %li= link_to t("devise.shared.links.sign_up"), new_user_registration_path, class: 'login'
-                  %li= link_to t('devise.shared.links.sign_in'), new_user_session_path, class: 'login'
+                = render 'layouts/header_not_connected'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.